multicluster/apis/multicluster/v1alpha2/clusterclaim_types.go

```diff
@@ -1,55 +0,0 @@
-/*
```

Comment:
To support existing API or ClusterSet, we should auto update ClusterSet with existing ClusterClaims?

Reply:
@jianjuns If we want to support existing ClusterClaim and update ClusterSet automatically, we need to keep ClusterClaim API. Do you mean we remove the required ClusterClaim CRs for ClusterSet setup and manifests, but keep the ClusterClaim API and clientset for backward capability? 
I feel it's OK to do so, but not sure how long we should maintain this API?

Reply:
Another concern is how to handle existing ClusterClaims when it has been auto-detected and updated it ClusterSet? I suppose it should be OK to delete it by MC controller? Otherwise, it may lead confusion and mis-configuration when customer tries to rebuild a ClusterSet with new cluster ID and ClusterSet ID in ClusterSet directly, the auto-detection code may rewrite and fail in a loop since the IDs are immutable.

Reply:
If we support auto-conversion, then we should not auto-delete existing ClusterClaims, but should let users delete them. The conversion code should check and not override Cluster ID/CluserSet ID in the ClusterSet CR if the two fields are already set. Yes, we should still update manifest and document to remove ClusterClaim and only talk about only ClusterSet.

Having said that, if we never know any serious users of Antrea Multi-cluster, I am fine to remove ClusterClaim without auto-conversion too.

Reply:
I added a ClusterClaim conversion controller to handle existing ClusterClaims.